tests/contrib/httplib/test_httplib.py

```diff
@@ -186,10 +187,19 @@ def test_httplib_request_get_request_https(self):
         # TODO: figure out how to use https in our local httpbin container
         conn = self.get_https_connection("httpbin.org")
         with contextlib.closing(conn):
-            conn.request("GET", "/status/200")
-            resp = conn.getresponse()
-            self.assertEqual(self.to_str(resp.read()), "")
-            self.assertEqual(resp.status, 200)
+            # Workaround for directly calling httpbin.org is flaky, so try multiple times to get proper 200 response
+            for i in range(50):
```

Comment:
```suggestion
            for i in range(5):
```

?

50 is a little aggressive no?

Reply:
Considering we're only sleeping 0.1 seconds per iteration, I think a max 5 seconds delay for this test is acceptable.

Reply:
we could also just sleep longer between retries. We don't want to DDOS `httpbin.org` since you know... we don't own that service.


we could also consider hitting another url, e.g. maybe `https://circleci.com/api/v2/project/gh/DataDog/dd-trace-py` has a high chance of being up/available during CI runs?

Reply:
Fair point. I'll decrease the number of retries and sleep longer for now. I think the long term solution here is to use another url, but for now I think this should suffice.